main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const carousel = document.querySelector('.carousel');
+const nextButton = document.querySelector('.button-next');
+const prevButton = document.querySelector('.button-previous');
+
+nextButton.addEventListener('click', () => {
+  carousel.scrollLeft += 390;
+
+  if (carousel.scrollLeft > 0) {
+    prevButton.classList.remove('button-disabled');
+    prevButton.classList.add('button-active');
+  }
+
+  if (carousel.scrollLeft === 910) {
+    nextButton.classList.remove('button-active');
+    nextButton.classList.add('button-disabled');
```

Comment:
toggle 1 class will be enough, because you do not have the third state and can add active styles to the general button styles